tasks/build/preconfigure-k3s-auto-deploying-manifests.yml

```
@@ -0,0 +1,14 @@
+---
+
+- name: ensure that the manifests directory exists, if required
+  file:
+    state: directory
+    path: "{{ k3s_server_manifests_dir }}"
+  when: k3s_server_manifests_templates | length > 0
+
+# https://rancher.com/docs/k3s/latest/en/advanced/#auto-deploying-manifests
+- name: Copy Auto-Deploying Manifests to Cluster
+  template:
+    src: "{{ item }}"
+    dest: "{{ k3s_server_manifests_dir }}/{{ item | basename | replace('.j2','') }}"
+  with_items: "{{ k3s_server_manifests_templates }}"
```

```suggestion
  loop: "{{ k3s_server_manifests_templates }}"

```

Typically I'd use `loop` rather than `with_items`.